lessons/04-structured-output/README.md

```diff
@@ -0,0 +1,590 @@
+
+The bright light from the Time Beetle fades, and you find yourself standing at the base of a massive stone pyramid. The air is thick with the scent of incense, and the distant sounds of chanting fill your ears. As you look around, you realize that you've landed in the heart of the Aztec Empire. The aerial screw, now a bit worse for wear, rests awkwardly on a large, intricately carved stone, with markings making you think it might be used as a calendar
+
+<div>
+  <img src="./assets/aztec.png" alt="Aztec calendar" width="400" />
+</div> 
+
+*Aztec calendar - Wikipedia* 
+
+Leonardo da Vinci steps forward, his eyes wide with wonder.
+
+**Leonardo:** "Incredible," he murmurs, running his fingers over the carvings. "But I hope this was of no importance." 
+
+You glance down and your heart sinks. The stone is part of the Aztec calendar, and the aerial screw has damaged a section of it. Before you can react, a group of Aztec soldiers approaches, their obsidian-tipped spears glinting menacingly in the sunlight. Their leader, a tall warrior with a feathered headdress, steps forward, his eyes narrowing as he takes in your strange attire and the damaged calendar.
+
+**Soldier leader** "Who are you, and what have you done?" he demands in Nahuatl. 
+
+<div>
+  <img src="./assets/meeting.png" alt="Meeting with Aztecs" width="400" />
+</div> 
+
+You take a deep breath, trying to steady your nerves. 
+
+**You:** "We are travelers from a distant land," you begin, hoping your knowledge of ancient languages will serve you well. "This is a flying machine, a gift from the gods to help us reach you. We did not mean to damage your sacred calendar.
+
+The warrior's eyes widen in surprise, but he remains cautious. 
+
+**Soldier leader:** "You will come with us to meet the ruler Montezuma. He will decide your fate.
+
+As you follow the soldiers towards the pyramid, you can't help but feel a mix of relief and apprehension. 
+
+Upon reaching the top of the pyramid, you are led into a grand chamber where Montezuma, the ruler of the Aztec Empire, sits on a throne adorned with gold and feathers. His piercing gaze falls upon you, and he gestures for you to approach.
+
+**Montezuma** "You claim to be travelers from a distant land," Montezuma says, his voice calm but authoritative. "And you have damaged our sacred calendar. How do you intend to make amends?"
+
+<div>
+  <img src="./assets/montezuma.jpeg" alt="Meeting with Montezuma" width="400" />
+</div> 
+
+<audio controls>
+  <source src="./assets/montezuma.mp3" type="audio/mpeg">
+  Your browser does not support the audio element.
+</audio>
+
+## Talk to Montezuma
+
+If you want to talk to Montezuma, run the app [Montezuma](./characters/). 
+
+> NOTE: Remember to start a Codespaces and then run `npm install` followed by `npm start`. 
```

Comment:
After they start the Codespace they'll need to navigate to lessons/04-structured-output/characters. Maybe change it to:

```
> NOTE: Remeber to start a GitHub Codespace, navigate to lessons/04-structured-output/characters, and then run `npm install` followed by `npm start`.